tests/test_forecast_live.py

```
@@ -0,0 +1,246 @@
+        for outcome in (True, False):
+            bids, asks = live_client._order_book_ladders(query_id, outcome)
+            best = live_client.get_best_prices(query_id, outcome)
+
+            for level in bids + asks:
+                assert 0 < level.price < 100, (
+                    f"price {level.price} outside the 1-99 cent convention"
+                )
+                assert level.size > 0
+
+            if bids and best["best_bid"] is not None:
+                assert max(l.price for l in bids) == best["best_bid"]
```

Ruff (0.16.0)

[error] Ambiguous variable name: `l`

(E741)